lib/build/geo_feature_builder.dart

```diff
@@ -86,12 +86,12 @@ class GeoFeatureBuilder implements Builder{
 
   Expression buildMultiPolygonInstance(Map<String, dynamic> geometry) {
```

Comment:
`buildMultiPolygonInstance` should not check what type it gets (MultiPolygon or Polygon) it should always only get the coordinates for a `MultiPolygon` like `buildPolygonInstance` and `buildRingInstance`. The actual checking for the type should happen outside in `buildGeoFeatureListInstance`

Reply:
The point is that regarding the type of the geometry I need to get a Multipolygon reference that is why I put it there